cluster/images/etcd/Makefile

```diff
@@ -26,8 +26,8 @@
 # Except from etcd-$(tag) and etcdctl-$(tag) binaries, we also
 # need etcd and etcdctl binaries for backward compatibility reasons.
 # That binary will be set to the last tag from $(TAGS).
-TAGS?=2.2.1 2.3.7 3.0.17
```

Comment:
Depends on supported versions of kube doesn't it.  What's the oldest supported version of kube?